Services/Style/classes/class.ilStyleDefinition.php

```diff
@@ -437,6 +441,28 @@ public static function getCurrentMasterStyle()
 		return $cs;
 	}
 
+	public static function getCurrentSkinVersion()
+	{
+		/** @var ilStyleDefinition $styleDefinition */
+        global $styleDefinition;
+
+        if (!isset(self::$current_skin))
+        {
+            self::getCurrentSkin();
```

Comment:
Does this line have any affect?

Reply:
No, not anymore. At the beginning I got the current skin manually, for testing, and later changed it. Seems that I forgot to remove the if clause. I will remove the if clause.